Gemfile

```
@@ -65,6 +65,8 @@ group :production do
 
   gem 'sentry-raven'
   gem 'sidekiq', '~> 5.2.1'
+
+  gem 'active_elastic_job'
```

That uses the old gem, not our gem.

```suggestion
  gem 'active_elastic_job', git: 'https://github.com/magara/active-elastic-job.git'
```